README.md

```diff
@@ -75,9 +79,9 @@ Select 1:
 Identify the three places CSS can go, and rank them in terms of specificity:
 
 ```text
-1.
-2.
-3.
+1. Inside of a Style.css file (most specific)
+2.  Inside of <style> tags in an HTML file (not horrible but could be more specific)
+3.Inline (least specific. Don't do this.)
 ```
```

Comment:
While an external stylesheet is the preferred method of writing out CSS, inline styling actually has the highest specificity. So if a property in a stylesheet and a property defined using inline styling conflict, the latter will override the former. This does not, however, mean that it's better.